packages/components/src/components/ExternalCard/elements.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+export const StyledWrapper = styled(Stack)<{ thumbnail: string }>`
+  position: relative;
+  overflow: hidden;
+  height: 156px;
+  width: 276px;
+  background: #999999;
+  border-radius: 4px;
+
+  &:after {
+    position: absolute;
+    content: '';
+    display: block;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background-image: url(${props => props.thumbnail});
+    background-size: cover;
+    background-position: center;
+    transform: scale(1);
+    transition: transform ease-in-out;
+    transition-duration: 100ms;
+  }
+
+  &:hover::after {
+    transform: scale(1.01);
+  }
+`;
+StyledWrapper.defaultProps = {
+  direction: 'vertical',
+};
+
+export const StyledContent = styled(Stack)`
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  display: flex;
+  background: linear-gradient(
+      0deg,
+      rgba(25, 25, 25, 0.3) 0%,
+      rgba(25, 25, 25, 0) 100%
+    ),
+    linear-gradient(0deg, rgba(25, 25, 25, 0.6) 0%, rgba(25, 25, 25, 0) 100%),
+    linear-gradient(0deg, rgba(25, 25, 25, 0.8) 0%, rgba(25, 25, 25, 0) 100%);
+`;
+
+export const StyledDetails = styled(Stack)`
+  margin: auto 24px 24px;
+`;
+StyledDetails.defaultProps = {
+  align: 'center',
+  gap: 3,
+  justify: 'space-between',
+};
+
+export const StyledTitle = styled(Text)<{ clamp?: boolean }>`
+  margin: 0;
+  color: #ebebeb;
+  font-size: 13px;
+  font-weight: 500;
+  line-height: 16px;
+
+  ${props =>
+    props.clamp &&
+    `  
+  width: 100%;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  `}
+`;
+StyledTitle.defaultProps = {
+  as: 'p',
```

Comment:
Because it's inside an `a` element, we should use `span`. I think this is the default for the `Text` component so we can remove this line. Also, the `margin: 0;` can be removed this way.

Reply:
@tristandubbeld is there a specific reason for using the `span`? There's no semantic impeditive to it, quoting the spec:

> The [a](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-a-element.html#the-a-element) element may be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g. buttons or other links).

Reference: https://www.w3.org/TR/2011/WD-html5-author-20110809/the-a-element.html.

Reply:
Hmm yeah you're right. It's interactive elements that aren't allowed as children. My reasoning was that since it's just a few words `p` wouldn't make much sense. But it's not wrong indeed, thanks for pointing this out!